README.md

```diff
@@ -30,10 +30,38 @@ pre-commit hook on every commit. To ignore words you can put them in the
 `codespell.txt` file. This file should list the supposedly misspelled words with
 one word per line and is case-sensitive.
 
-## Exercises guidelines
+## C++ guidelines
 
 - Prefer "east const" for new stuff.
-- For the time being, maintain both a `Makefile` and a `CMakeLists.txt`.
-- Each exercise should have a `solution` subdirectory.
-- `CheatSheet` file(s) is/are a roadmap for mentors.
-- Instructions for students stay in `README.md` files and/or directly in the main exercise file.
+
+## Exercises guidelines
+
+### About instructions for mentors and students
+
+Cheat sheet is only for mentors, with roadmap, solutions and key points to discuss.
+
+Readmes are for stundents. They hosts the "basic building instructions, gives the name of the main program file, and possibly gives a
+summary of the exercise goals and steps.
+
+Inside the code, one get more precise help on what should/could be done and also where (when the code is long and only one part needs to be worked on).
```

Comment:
Maybe split this in two sections:
```suggestion
### Instructions for students

Look at the readmes. They host the basic building instructions, give the name of the main program file, and possibly give a summary of the exercise goals and steps.

Inside the code, one gets more precise help on what should/could be done and also where (e.g. when the code is long and only one part needs to be worked on).

### Instructions for mentors

There is a [cheat sheet](code/ExercisesCheatSheet_All.md) with roadmap, solutions and key points to discuss.
```

Reply:
Your suggestion makes me wonder : do you think this global README will be read by students ? My feeling was that, at least as it is written today, it was only done for mentors and material contributors. If we expects student to read, a larger refactoring would be needed, IMO.

Reply:
I do think this is read by the students. We tell them to clone this repository for the exercises and we also have links here in the slides.

But I can now understand your PR better.

Reply:
Sebastien initially put here some information only for contributors. Time has come, I guess, to make a difference. I fully agree that this README is the first place where students will go. Actually, I would make a different README-contributors.md dedicated to contributors.